simple_network_sim/network_of_populations.py

```diff
@@ -31,40 +33,7 @@ class NetworkOfPopulation(NamedTuple):
     movementMultipliers: Dict[Time, loaders.Multiplier]
     infectiousStates: List[Compartment]
     infectionProb: Dict[Time, float]
-
-
-# NotCurrentlyInUse
```

Comment:
Looks like we won't use it anymore? I can add it back if necessary

Reply:
I suggest we don't do changes like this in the same PR as other changes. This will cause a conflict with another change where I turned this into a package, for instance.

Reply:
ok, reverted the change